romancal/conftest.py

```diff
@@ -261,3 +261,41 @@ def base_image():
     """
 
     return _base_image
+
+
+@pytest.fixture
+def ignore_metadata_paths():
+    """
+    List of metadata paths that will contain always variable values.
+
+    These include versions, dates, logs, etc. that will often differ.
+    """
+    return [
+        "asdf_library",
+        "history",
+        "roman.meta.ref_file.crds.version",
+        "roman.meta.calibration_software_version",
+        "roman.cal_logs",
+        "roman.meta.cal_logs",
+        "roman.meta.date",
+        "roman.meta.file_date",
+        "roman.individual_image_cal_logs",
+        "roman.meta.individual_image_meta",
+    ]
+
+
+@pytest.fixture
+def ignore_parquet_metadata_paths(ignore_metadata_paths):
+    """
+    List of parquet metadata paths to ignore during testings.
+    """
+    return [
+        *ignore_metadata_paths,
+        "table_meta_yaml",
+        "source_catalog",
+        "roman.meta.filename",
+        "roman.meta.model_type",
+        "roman.meta.ref_file",
+        "roman.meta.image",
+        "roman.meta.forced_segmentation",
```

Comment:
Just trying to remember these... table_meta_yaml doesn't obviously vary to me but it's not really "official" metadata and I'm happy to ignore it.  source_catalog has some dates in it and so it makes sense to ignore it.  I'm mildly surprised to see filename here but not under the asdf variant.  image includes file dates and so it makes sense.  model_type and to a lesser extent forced_segmentation I don't understand well.

Reply:
These are for comparing the input datamodel metadata (for example MosaicModel) with the parquet metadata (produced from the MosaicSourceCatalogModel). So things like:
- `table_meta_yaml` will never be in the datamodel metadata
-`meta.filename` will always differ
- `meta.image` and `meta.forced_segmentation` will never be in the datamodel metadata
- `meta.model_type` will always differ